tests/core/test_search.py

```diff
@@ -256,51 +258,170 @@ def test_create_vespa_search_params(
+        "family_ids,document_ids"
+    ),
+    [
+        (
+            True,
+            (1940, 1960),
+            None,
+            SortOrder.DESCENDING,
+            None,
+            10,
+            10,
+            0,
+            "ABC",
+            None,
+            None,
+        ),
+        (
+            False,
+            (1940, None),
+            None,
+            SortOrder.DESCENDING,
+            {
+                FilterField.COUNTRY: ["germany", "France"],
+                FilterField.REGION: ["europe"],
+            },
+            20,
+            10,
+            0,
+            "ABC",
+            ["CCLW.document.1.0", "CCLW.document.2.0"],
+            None,
+        ),
+        (
+            False,
+            (1940, None),
+            None,
+            SortOrder.DESCENDING,
+            {
+                FilterField.COUNTRY: ["germany", "France"],
+                FilterField.REGION: ["europe"],
+            },
+            20,
+            10,
+            0,
+            "ABC",
+            ["CCLW.executive.1.0", "CCLW.executive.2.0"],
+            ["CCLW.document.1.0", "CCLW.document.2.0"],
+        ),
+    ],
+)
+def test_create_browse_request_params(
+    test_db,
+    exact_match,
+    year_range,
+    sort_field,
+    sort_order,
+    keyword_filters,
+    max_passages,
+    limit,
+    offset,
+    continuation_token,
+    family_ids,
+    document_ids,
+):
+    db_setup(test_db)
 
-    return geo_names
+    SearchRequestBody(
+        query_string="",
+        exact_match=exact_match,
+        max_passages_per_doc=max_passages,
+        family_ids=family_ids,
+        document_ids=document_ids,
+        keyword_filters=keyword_filters,
+        year_range=year_range,
+        sort_field=sort_field,
+        sort_order=sort_order,
+        include_results=None,
+        limit=limit,
+        offset=offset,
+        continuation_token=continuation_token,
+    )
 
 
 @pytest.mark.parametrize(
-    "filters",
+    "filters, expected",
     [
-        None,
-        {FilterField.CATEGORY: ["Executive"]},
-        {FilterField.LANGUAGE: ["english"]},
-        {FilterField.SOURCE: ["CCLW"]},
-        {FilterField.REGION: ["europe-central-asia"]},
-        {FilterField.COUNTRY: ["france", "germany"]},
-        {FilterField.COUNTRY: ["cambodia"]},
-        {FilterField.REGION: ["south_america"], FilterField.COUNTRY: ["france"]},
+        (None, None),
+        ({FilterField.REGION: ["this-is-not-a-region"]}, None),
+        (
+            {
+                FilterField.REGION: ["latin-america-caribbean"],
+                FilterField.COUNTRY: ["france"],
+            },
+            None,
+        ),
+        ({FilterField.REGION: ["north-america"]}, {"family_geography": ["CAN", "USA"]}),
+        (
+            {
+                FilterField.REGION: ["north-america"],
+                FilterField.COUNTRY: ["not-a-country"],
+            },
+            {"family_geography": ["CAN", "USA"]},
```

Comment:
Is this correct based off of the and condition? I.e. shouldn't we be returning None as the country doesn't exist in north america?

Reply:
Current behaviour is to ignore countries that don't exist, theres not a need for anything better than this right now because the frontend only allows specific values. Not sure what we want longer term, maybe an error message saying if a country filter doesnt exist, but I think it needs more thought